examples/vue/dependent-queries/src/Post.vue

```diff
@@ -0,0 +1,66 @@
+<script lang="ts">
+import { computed, defineComponent } from "vue";
+import { useQuery } from "@tanstack/vue-query";
+
+import { Author, Post } from "./types";
+
+const fetchPost = async (id: number): Promise<Post> =>
+  await fetch(`https://jsonplaceholder.typicode.com/posts/${id}`).then(
+    (response) => response.json()
+  );
+
+const fetchAuthor = async (id: number): Promise<Author> =>
+  await fetch(`https://jsonplaceholder.typicode.com/users/${id}`).then(
+    (response) => response.json()
+  );
+
+export default defineComponent({
+  name: "PostDetails",
+  props: {
+    postId: {
+      type: Number,
+      required: true,
+    },
+  },
+  emits: ["setPostId"],
+  setup(props) {
+    const { isLoading, isError, isFetching, data: post, error } = useQuery(
+      ["post", props.postId],
+      () => fetchPost(props.postId)
+    );
+
+    const authorId = computed(() => post.value?.userId);
+
+    const { data: author } = useQuery(
+      ["author", authorId.value],
```

Comment:
```suggestion
      ["author", authorId],
```
You should not unwrap reactive value here, otherwise it will stay under the same key, as in vue `setup` function is run only once.

Reply:
Thank you, updated the code.